packages/forms/src/UIForm/fields/FieldTemplate/FieldTemplate.scss

```diff
@@ -28,3 +28,18 @@
 .updating {
 	@include heartbeat(object-blink);
 }
+
+.field-template-title {
+	display: flex;
+	align-items: center;
+
+	:global {
+		.control-label {
+			margin-right: 5px;
```

Comment:
Suggestion: You can add a margin-left on the btn class instead to avoid a new classname